src/main/java/core/basesyntax/entity/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.entity;
+
+public class IsoscelesTrapezoid extends Figure {
+    public static final int MAX_HIGH = 20;
+    public static final int MAX_UPPER_BASE = 20;
+    public static final int MAX_LOWER_BASE = 20;

```

Comment:
why here is `int` types and for default values is a `double`. It is better to use double everywhere for consistency. Check other classes please.